### ISSUE TITLE
nvda causing the outlook security message to be displayed

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7953]
After updating nvda to update next 14828, after opening outlook 2013 and focusing a message in any box, outlook displays an error message stating:
Progr. trying to access inform. of email address
stored in Outlook. If this is unexpected, click
deny and make sure your antivirus is up to date.
For more information. about email security and how to
To prevent this warning, click Help.
Allow access by
1 minute
Allow Deny Help
I have read in the Outlook Help that it is possible to disable these warnings. However, this machine is a corporate machine and when trying to access the option to unblock the security warnings I am prevented from changing due to the security policies of the company where I work.
Here is the path that is oriented in the Microsoft article that I can not access:
View security settings in the Trust Center
The Schedule Access security settings in the Trust Center provide the following options:
•
Warn about suspicious activity when my antivirus software is down or outdated (recommended) This is the default setting in Outlook. Suspicious activity refers to an untrusted program that is trying to access Outlook.
•
Always warn about suspicious activity This is the most secure setting and you are always prompted to make a trusted decision when a program tries to access Outlook.
•
Never warn about suspicious activity (not recommended) This is the least secure setting.
To view these settings, follow these procedures:
1.
Click the File tab.
2.
Click Options.
3.
Click Trust Center.
4.
Click Trust Center Settings.
5.
Click Program Access.
Comments:
•
If your computer is managed by a Microsoft Exchange Server administrator or a Microsoft Windows Active Directory Domain Services administrator in your organization, and the administrator changes the default setting and prevents users from making changes, the option to customize Access security settings will be disabled.
In the Outlook warning window I can change the security warning only for 10 minutes, which makes using nvda with outlook extremely terrible, especially during work hours.
Please, please provide to nvda an option to return to the previous behavior where these warnings were not displayed.
Or undo the changes that caused this outlook behavior to be undone.
With the current behavior, can postpone the warnings only for 10 minutes, and can not disable the security warnings of outlook due to the security policies of the company where work is almost unbearable to use nvda.
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
Using the nvda next 14828 open a message box with emails, and put the arrows on a message.
In outlook the warnings of a program trying to access the contact information must be enabled.

## Expected behavior:
Nvda should not cause outlook security messages to be shown.

### Actual behavior:
Nvda causes outlook security messages to be displayed.

### System configuration:
NVDA version:
> Example: next-14828.

NVDA Installed or portable:
instaled

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 7

Name and version of other software in use when reproducing the issue:
outlook 2013
### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
> Please list them and the result
yes
next 14811 
this problem does not occur.

**Comment by leonardder** [comment]
cc @dkager

**Comment by leonardder** [comment]
Hmm, this could be caused by #7949. It would be a real drawback if that's the case.

**Comment by leonardder** [comment]
@fernando-jose-silva: Would you be able to test the two builds below?

* [Try build](https://ci.appveyor.com/api/buildjobs/h3risx3r3mj8lp11/artifacts/output%2Fnvda_snapshot_try-outlookReplyStatus-14832%2C673dc9c3.exe)
* [Master snapshot](https://ci.appveyor.com/api/buildjobs/b03x47mw64sm91wv/artifacts/output/nvda_snapshot_master-14831,3ff4700d.exe)

If the try build does have the security question problem and the master snapshot does not, this is indeed caused by #7949.

**Comment by leonardder** [comment]
You could also verify this as follows:

1. Reply to a message in Outlook.
2. When you arrow over the message you have replied to, NVDA should say replied before it announces the sender, subject, etc. If the issue is caused by the recent pr, NVDA shouldn't give this information when you denied access, and it should report this information when you allowed access.

**Comment by PratikP1** [comment]
The problem appears to be limited to versions of Outlook 2013 and below. 2016 seems to be fine.

**Comment by leonardder** [comment]
[**@PratikP1**](https://github.com/PratikP1) commented on [2 Feb 2018, 20:04 CET](https://github.com/nvaccess/nvda/issues/7953#issuecomment-362675891 "2018-02-02T19:04:36Z - Replied by Github Reply Comments"):
> The problem appears to be limited to versions of Outlook 2013 and below.

Where are these findings based on?

**Comment by PratikP1** [comment]
Correction: I shouldn't have assumed "2013 and below." So far as reported, the problem is with 2013. I has only tested 2016 and the problem doesn't occur in Outlook 2016.

**Comment by leonardder** [comment]
This is most definitely caused by #7949, and I've pushed another commit to the associated branch that will use a completely different implementation to retrieve forwarded/replied state.

**Comment by leonardder** [comment]
The code causing this has been backed out of next, so closed this.
